Bottom: d22cbcf9f2efdeea42a047c637cddeb78bc52d10
Top:    df8b678743aabcdbdf009dc9e66455fddb305e8a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-08-04 10:01:57 -0700

Refresh of idx-err

---

diff --git a/src/indexer.c b/src/indexer.c
index b9e400b..f9042f5 100644
--- a/src/indexer.c
+++ b/src/indexer.c
@@ -35,11 +35,11 @@
 #  include <config.h>
 #endif /* HAVE_CONFIG_H */
 
+#include <errno.h>
 #include <sys/types.h>
 #include <stdlib.h>
 
 #include "indexer.h"
-#include "cma.h"
 
 /*
  * Indexer - to find a structure given an index
